pom.xml

```diff
@@ -85,6 +85,16 @@
             <version>1.7.12</version>
         </dependency>
         <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.1.11</version>
+        </dependency>
+        <dependency>
```

Comment:
@sanjeevchopra, we tested with logback. Anyone using log4j, may have to exclude logback dependencies explicitly. Hope, FrankGasparovic can do further integration test after Pull.